libbeat/processors/cache/config.go

```diff
@@ -88,19 +88,24 @@ func defaultConfig() config {
 }
 
 type storeConfig struct {
-	Memory *id `config:"memory"`
-	File   *id `config:"file"`
+	Memory *memConfig  `config:"memory"`
+	File   *fileConfig `config:"file"`
 
 	// Capacity and Effort are currently experimental
 	// and not in public-facing documentation.
 	Capacity int `config:"capacity"`
 	Effort   int `config:"eviction_effort"`
 }
 
-type id struct {
+type memConfig struct {
 	ID string `config:"id"`
 }
 
+type fileConfig struct {
+	ID            string        `config:"id"`
+	WriteOutEvery time.Duration `config:"write_frequency"`
```

Comment:
I need to look over the [rest of the code base](https://github.com/search?q=repo%3Aelastic%2Fbeats%20path%3A**%2F*.go%20%2Ftime.Duration%20%2B%60config%3A%22%2F&type=code) to see if there is any consistency for naming this type of setting. My initial thought is that this is an "interval" rather than a "frequency", because an interval is a specified as duration between events.